QRScanner/QRScannerView.swift

```diff
@@ -241,23 +270,25 @@ public class QRScannerView: UIView {
             strongSelf.qrCodeImageView.frame = path.bounds
             strongSelf.qrCodeImageView.center = center
             }, completion: { [weak self] _ in
+                self?.qrCodeImageView.image = self?.qrCodeImage
                 self?.delegate?.success(qrCode)
```

Comment:
[IMO] This is the Weak/Strong Dance, it have some risks.

the Weak/Strong Dance 
@c : http://kelan.io/2015/the-weak-strong-dance-in-swift/

```
guard let strongSelf = self else { return }
strongSelf.qrCodeImageView.image = self?.qrCodeImage
strongSelf.delegate?.success(qrCode)
```

Reply:
Oh, I did not know that.
OK thanks!